translator/reader/read_geometry.cpp

```diff
@@ -91,7 +91,13 @@ void UsdArnoldReadMesh::Read(const UsdPrim &prim, UsdArnoldReaderContext &contex
     }
 
     // Vertex positions
-    ReadArray<GfVec3f, GfVec3f>(mesh.GetPointsAttr(), node, "vlist", time);
+    UsdAttribute pointsAttr = mesh.GetPointsAttr();
+    ReadArray<GfVec3f, GfVec3f>(pointsAttr, node, "vlist", time);
```

Comment:
The return value of ReadArray is not used anywhere in the code, maybe we could return the number of keys instead of array length, and use that to check if the attribute is animated?

Documentation of ValueMightBeTimeVarying:

```
    /// Return true if it is possible, but not certain, that this attribute's
    /// value changes over time, false otherwise. 
    ///
    /// If this function returns false, it is certain that this attribute's
    /// value remains constant over time.
```

Reply:
yep, makes sense. Also, ReadArray not only checks if the prim attribute might be varying, but also if we have motion blur in the reader options. This is more consistent now